CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ ENDIF()
 
 IF(WITH_VIGRANUMPY)
     set(WITH_BOOST "ON")
-    # set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
+    set(WITH_BOOST_COMPONENTS ${WITH_BOOST_COMPONENTS} "python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}")
```

Comment:
In theory, you are right that this is the best approach. I tried it before resorting to `FIND_LIBRARY` and found it to be non-portable, because the `python-py*` naming convention is not universally followed (I forgot who caused problems).